pyramid_swagger/tween.py

```diff
@@ -108,10 +115,11 @@ def validator_tween(request):
                 return handler(request)
 
         if settings.validate_request:
-            request_data = handle_request(
+            request_data = swagger_handler.handle_request(
                 PyramidSwaggerRequest(request, route_info),
-                validation_context,
-                validator_map)
+                validation_context=validation_context,
+                validator_map=op_or_validators_map,
+                op=op_or_validators_map)
```

Comment:
This is passing the same argument in twice, why not just use positional args and pass it in once?